### ISSUE TITLE
Binary Classifier experiment - log loss

### ISSUE THREAD

**Issue by andrewcz** [uci-cbcl/DanQ#2]
Hi,
Fantastic Library!
I was just wondering, i am trying to use the library for a binary classifier experiment using the log loss function to train the model. This is for a university experiment around benchmarking different models. Would you have time to provide an example of how to use the library to achieve the above goal.
Also, with an included visualisation on how the model learns.
Many thanks,
Best,
Andrew

**Comment by daquang** [comment]
Hi Andrew!

Sorry for the delay in response. I am glad you like the library. Yes, I would love to provide an example. Can you give me more details? Perhaps you can share with me the sample training data and labels?

Thanks,
Daniel

**Comment by andrewcz** [comment]
Hi,
Thank you!
Il prepare some data and code and post it this afternoon.
Many thanks,
Best,
Andrew
